setup.sh

```diff
@@ -70,6 +70,7 @@ pssh --inline \
     --host "$MASTERS $SLAVES" \
     --user root \
     --extra-args "-t -t $SSH_OPTS" \
+    --timeout 300 \
```

Comment:
It's a 60 second timeout by default. I did have it set to 0 before but @Shivaram suggested that that might not be optimal

—
Best Wishes,
Tomer Kaftan

On Mon, Jan 26, 2015 at 7:12 PM, Nicholas Chammas
<notifications@github.com> wrote:

>> @@ -70,6 +70,7 @@ pssh --inline \
>>      --host "$MASTERS $SLAVES" \
>>      --user root \
>>      --extra-args "-t -t $SSH_OPTS" \
>> +    --timeout 300 \
> @tomerk What timeout do you see without this setting? The man page doesn't mention a default value.
> It does mention, though, that you can set the timeout to 0 and pssh will wait until the called processes complete. Does that not work here?
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/mesos/spark-ec2/pull/94/files#r23584687